met-etl/migrations/versions/ed925c588835_initial_version.py

```diff
@@ -0,0 +1,39 @@
+"""initial version
+
+Revision ID: ed925c588835
+Revises: 
+Create Date: 2022-07-08 11:38:31.024195
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ed925c588835'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('survey',
```

Comment:
Not sure I understand the analytics db model yet, I'm guessing this would be so it has access to the survey name?

Reply:
this is totally dummy..I will remove this when i work on the actual data model ..just wanted to see if cron can fetch data